test/FastTests/Corax/LookupBulkTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FastTests.Voron;
+using FastTests.Voron.FixedSize;
+using Voron.Data.Lookups;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Corax;
+
+public class LookupBulkTests : StorageTest
+{
+    public LookupBulkTests(ITestOutputHelper output) : base(output)
+    {
+    }
+    [Theory]
+    [InlineData(214)]
+    [InlineDataWithRandomSeed]
+    public void CanBulkAddToEmptyLookup(int seed)
+    {
+        var random = new Random(seed);
+        var items = Enumerable.Range(0, 1024)
+            .Select(_ => new Int64LookupKey(random.NextInt64()))
+            .OrderBy(x=>x.Value).ToArray();
+
+        var values = new long[items.Length];
+        var offsets = new int[items.Length];
+
+        using (var wtx = Env.WriteTransaction())
+        {
+            var lookup = wtx.LookupFor<Int64LookupKey>("test");
+            lookup.InitializeCursorState();
+            
+            int wrote = 0;
+            while (wrote < items.Length)
+            {
+                var curItems = items[wrote..];
+                var curValues = values[wrote..];
+                var curOffsets = offsets[wrote..];
+                var read = lookup.BulkUpdateStart(curItems, curValues, curOffsets, out var pg);
+
+                Assert.Equal(curItems.Length, read);
+                Assert.NotEqual(0, pg);
+
+                var changed = lookup.CheckTreeStructureChanges();
+                int adjustment = 0;
+                for (int i = 0; i < read && changed.Changed == false; i++)
+                {
+                    lookup.BulkUpdateSet(ref curItems[i], 1 + wrote, pg, curOffsets[i], ref adjustment);
+                    wrote++;
+                }
+            }
+            
+            wtx.Commit();
+        }
+
+        using (var rtx = Env.ReadTransaction())
+        {
+            var lookup = rtx.LookupFor<Int64LookupKey>("test");
+            var it = lookup.Iterate();
+            it.Reset();
+            for (int i = 0; i < items.Length; i++)
+            {
+                Assert.True(it.MoveNext(out Int64LookupKey k, out var v, out _));
+                Assert.Equal(items[i].Value, k.Value);
+                Assert.Equal(i+1, v);
+            }
+            Assert.False(it.MoveNext(out Int64LookupKey _, out _, out _));
+
+        }
+    }
+    
+    [Theory]
+    [InlineData(214)]
+    [InlineDataWithRandomSeed]
+    public void CanBulkUpdateAndInsertAndRemove(int seed)
+    {
+        int count = 0;
```

Comment:
Can you please check the warning?

Reply:
Done in https://github.com/ravendb/ravendb/pull/17176/commits/8e7e924f5fbc03ee565615da251807af49a82a41